rpc/src/lib.rs

```diff
@@ -67,7 +67,10 @@ extern crate parity_version as version;
 extern crate rlp;
 extern crate stats;
 extern crate keccak_hash as hash;
+
+#[cfg(any(target_os = "linux", target_os = "macos", target_os = "windows"))]
```

Comment:
Removed android for now because I am not sure whether android compiles or not. 

I need to grab the `android NDK` in order to test it!

Reply:
The easiest solution is docker.
Go to `parity/docker/android` and create the docker image: `docker build -t parity-android .`
You can then compile Parity with it: `docker run --rm -it -v `pwd`:/usr/code -w /usr/code parity-android` and `cargo build`.

Reply:
Thanks Pierre!